shell/platform/darwin/ios/framework/Source/FlutterPlatformViewsTest.mm

```diff
@@ -2654,8 +2654,9 @@ - (void)testFlutterClippingMaskViewPoolReuseViewsAfterRecycle {
   CGRect newRect = CGRectMake(0, 0, 10, 10);
   FlutterClippingMaskView* view3 = [pool getMaskViewWithFrame:newRect];
   FlutterClippingMaskView* view4 = [pool getMaskViewWithFrame:newRect];
-  XCTAssertEqual(view1, view3);
-  XCTAssertEqual(view2, view4);
+  // view3 and view4 should randomly get either of view1 and view2.
+  XCTAssertEqual(std::set<FlutterClippingMaskView*>({view1, view2}),
+                 std::set<FlutterClippingMaskView*>({view3, view4}));
```

Comment:
May have to do `XCTAssertObjectEqual` and `NSSet`.

Reply:
interesting - did this pass the test? 

Probably because internally `XCTAssertEqual` simply does `==`, which happens to be custom defined by `std::set`, despite that `XCTAssertEqual` intends to do simple pointer comparison. So I still think it's better to do NSSet comparison.